### PR TITLE
Feature/91 cas6 UI

### LINKED ISSUE #91: CAS6: Adapt UI to CES design
For CAS6, the Cloudogu theme is to be used. For this purpose, the UI of CAS6 is to be adapted.
Accessibility is to be taken into account.

### PR DESCRIPTION
**Pull request by CloudFlo2312**
Resolves #91

### CODE REVIEW

File: app/bower.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (Subaro) [inline app/bower.json:1]: Is this bower.json still required? Not in my eyes. We should remove it.

File: app/.gitignore [hunk 2]
```diff
@@ -20,3 +20,4 @@ tmp/
 ./apache-tomcat
 apache-tomcat.zip
 config-metadata.properties
+/src/main/resources/bower_components/
```
Comment (Subaro) [inline app/.gitignore:23]: We should not use any bower components in CAS 6 so we should remove this entry.

File: app/build.gradle [hunk 3]
```diff
@@ -87,6 +87,7 @@ dependencies {
     implementation 'org.mousio:etcd4j:2.11.0'
     implementation 'com.googlecode.json-simple:json-simple:1.1.1'
     implementation 'org.apache.logging.log4j:log4j-core:2.14.1'
+    implementation 'org.webjars.bowergithub.cloudogu:ces-theme:0.2.1'
```
Comment (Subaro) [inline app/build.gradle:90]: Today I saw that a new version was released? We should consider upgrading as well.

File: app/src/main/resources/messages.properties [hunk 4]
```diff
@@ -0,0 +1,526 @@
+screen.welcome.welcome=Congratulations on bringing CAS online!  To learn how to authenticate, please review the default authentication handler configuration.
```
Comment (Subaro) [inline app/src/main/resources/messages.properties:1]: Is there any reason to overlay the messages.properties?

File: app/src/main/resources/messages_de.properties [hunk 5]
```diff
@@ -0,0 +1,296 @@
+Welcome Screen Messages GERMAN
```
Comment (Subaro) [inline app/src/main/resources/messages_de.properties:1]: See above

File: app/src/main/resources/templates/casLoginView.html [hunk 6]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```
Comment (Subaro) [inline app/src/main/resources/templates/casLoginView.html:1]: How does the screenreader understand the views? Is there any problem? We should consider accessibility as stated in the story.

File: app/src/main/resources/templates/fragments/pmlinks.html [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+    <form th:if="${passwordManagementEnabled}" method="post" id="passwordManagementUserForm">
+        <input type="hidden" name="execution" th:value="${flowExecutionKey}"/>
+        <input type="hidden" name="_eventId" value="forgotUsername"/>
+        <span class="fa fa-unlock"></span>
+        <a id="forgotUsernameLink" th:utext="#{screen.pm.button.forgotUsername}" href="javascript:void(0)"
+           onclick="document.getElementById('passwordManagementUserForm').submit();"/>
+    </form>
+
+    <script>
+        console.log("Loaded scripts");
```
Comment (Subaro) [inline app/src/main/resources/templates/fragments/pmlinks.html:36]: We should remove this debug code

File: docs/operations/development/Setup_Integrationtests_en.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
+# Setup for the integration tests
+
+This section describes the steps required to properly run the integration tests.
+
+## Requirements
+
+* It is necessary to install the program `yarn
```
Comment (Subaro) [inline docs/operations/development/Setup_Integrationtests_en.md:7]: The \`yarn misses an \`

File: docs/operations/development/Setup_Integrationtests_en.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+
+**Step 3**
+
+In order for our tests for the password forgetting functionality to be carried out, we must define a text in the CAS that is to be displayed when the password forgetting button is clicked.
+A corresponding entry in etcd can be configured in the following way:
+
+```bash
+etcdctl set /config/cas/forgot_password_text 'Contact your admin'
+```
+
```
Comment (Subaro) [inline docs/operations/development/Setup_Integrationtests_en.md:81]: The entries for the legal links are not described in english

File: integrationTests/cypress/integration/legal_urls.feature [hunk 10]
```diff
@@ -0,0 +1,6 @@
+Feature: Simple test to check the forgotten password functionality of cas
+
+  Scenario: the user successfully logs into the CES with admin credentials and CAS shows the profile page
```
Comment (Subaro) [inline integrationTests/cypress/integration/legal_urls.feature:3]: Please adapt the scenario description to the actual test

File: integrationTests/cypress/integration/legal_urls.feature [hunk 11]
```diff
@@ -0,0 +1,6 @@
...
+    Given the user is currently not logged in the CES
+    Given the browser shows the cas login page
```
Comment (Subaro) [inline integrationTests/cypress/integration/legal_urls.feature:5]: Gerkin tests without Given/When/Then steps seem kinda awkward. Maybe change the step `Given the browser shows the CAS login page` into `When the user visits the CAS login page`

Please spell CAS big as it is an abbreviation for a name.

File: integrationTests/cypress/integration/password_forgotten.feature [hunk 12]
```diff
@@ -0,0 +1,7 @@
+Feature: Simple test to check the forgotten password functionality of cas
+
+  Scenario: the user successfully logs into the CES with admin credentials and CAS shows the profile page
```
Comment (Subaro) [inline integrationTests/cypress/integration/password_forgotten.feature:3]: Please adapt the scenario description to the actual test

File: integrationTests/cypress/integration/password_forgotten.feature [hunk 13]
```diff
@@ -0,0 +1,7 @@
...
+    Given the user is currently not logged in the CES
+    Given the browser shows the cas login page
```
Comment (Subaro) [inline integrationTests/cypress/integration/password_forgotten.feature:5]: Gerkin tests without Given/When/Then steps seem kinda awkward. Maybe change the step `Given the browser shows the CAS login page` into `When the user visits the CAS login page`

Please spell CAS big as it is an abbreviation for a name.

File: integrationTests/cypress/integration/legal_urls.feature [hunk 14]
```diff
@@ -0,0 +1,6 @@
+Feature: Simple test to check the forgotten password functionality of cas
```
Comment (Subaro) [inline integrationTests/cypress/integration/legal_urls.feature:1]: Wrong feature description

File: app/.gitignore (hunk 2 above)
Comment (Phil-Ah) [reply app/.gitignore]: removed from .gitignore

File: app/bower.json (hunk 1 above)
Comment (Phil-Ah) [reply app/bower.json]: bower.json removed.

File: docs/operations/development/Setup_Integrationtests_en.md (hunk 8 above)
Comment (Subaro) [reply docs/operations/development/Setup_Integrationtests_en.md:7]: Still missing the `

**Review by Subaro — CHANGES_REQUESTED** [review]

**Review by Subaro — APPROVED** [review]

**Pull request merged**